### ISSUE TITLE
Goal Information in Get-YnabCategory

### ISSUE THREAD

**Issue by hershsa** [ConnorGriffin/Posh-YNAB#20]
I'm trying to pull goal information along with the category. I added the following to the JsonParsing file, anything else you can think of or that I might have missed to get this to work?

GoalType = $_.goal_type
GoalCreationMonth = $_.goal_creation_month
GoalTarget = $_.goal_target
GoalTargetMonth = $_.goal_target_month

**Comment by ConnorGriffin** [comment]
Yep that looks good. I made the change and pushed it to master, you can update to `0.2.4` to see the change.

We may want to change the wording from TB/TBD/MF to Target Balance/Target Balance by Date/Monthly Funding to make it more clear to the user, but I'll leave it as is for now.

**Comment by hershsa** [comment]
Thanks! Looks great! Not a question specific about your module but your process for updating and testing locally. I've had a hard time getting it recognize when I make tweaks locally, how do you get powershell to "reload" the module when you've made changes to some of the functions?

**Comment by hershsa** [comment]
Scratch that, remove and import is working now that I'm editing the right files.

**Comment by ConnorGriffin** [comment]
Just as an FYI, I usually do this when testing. `-Force` automatically removes and reimports the module which saves you some time:  `Import-Module .\Posh-YNAB.psd1 -Force`

**Comment by hershsa** [comment]
Thanks!

I've expanded on Get-YnabCategory to work with the following endpoint - /budgets/{budget_id}/months/{month}/categories/{category_id}

Would be willing to contribute if you want. Not the greatest but should be able to piece stuff together to help out a little. I can either open an issues and put in what I did or contribute directly. Attempted to create a branch but guessing I need to be granted access for that to work.

Thanks for getting this to where it is today! Creating reports with the goal data is what I've been missing since moving to YNAB.
